spec/models/active_shipping_calculator_spec.rb

```diff
@@ -97,12 +87,8 @@
   describe 'compute' do
     subject { calculator.compute(package) }
 
-    let(:rates) do
-      [double('rate', service_name: 'Bogus Calculator', price: 999)]
-    end
-
     it 'should use the carrier supplied in the initializer' do
-      expect(carrier).to receive(:find_rates) { response }
+      expect(calculator.carrier).to be_an_instance_of(carrier.class)
```

Comment:
I don't see any value in this test, I think we can remove it altogether. The original test did nothing anyways, and we know that the carrier will be an instance of the carrier class, so no need for this I think?